### DISCUSSION TITLE
System Issues. Problem with downloading agent

### DISCUSSION THREAD

**Discussion by targetss** [amidaware/tacticalrmm#1651] [General]
- OS: Debian GNU/Linux 10 (buster)
 - Chrome (Version 117.0.5938.149 (Official build))
 - RMM Version: v0.16.4

**Installation Method:**
  - Standard

### Problem:
After the upgrade by staff resources started a problem with downloading a new installation file to install the RMM agent. This problem occurs when selecting any parameters for the agent. Only the file "exe" creates a problem, powershell script is downloaded without problems
Object request: 
{
    "installMethod": "exe",
    "client": 1,
    "site": 1,
    "expires": 24,
    "agenttype": "workstation",
    "power": 1,
    "rdp": 1,
    "ping": 1,
    "goarch": "amd64",
    "api": "https://api.rm.mydomain.ru",
    "fileName": "trmm---workstation-amd64.exe",
    "plat": "windows"
}

### Debug log (Error):
HTTPSConnectionPool(host='agents.tacticalrmm.com', port=443): Max retries exceeded with url: /api/v2/exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fd16dd14850>: Failed to resolve 'agents.tacticalrmm.com' ([Errno -3] Temporary failure in name resolution)"))

### Screen:
https://ibb.co/Nx792x9

**Comment by wh1te909** [comment]
Hi the error is right there in the debug log. your server cannot resolve DNS so just fix your DNS on the server

**Reply by targetss** [reply]
Oh, Thanks))
